app/lib/task/backend.dart

```diff
@@ -973,13 +973,21 @@ class TaskBackend {
   Future<String?> taskLog(String package, String version) async {
     final data = await gzippedTaskResult(package, version, 'log.txt');
     if (data == null) {
-      return null;
+      final status = await packageStatus(package);
+      if (status.runtimeVersion == null) {
+        return 'no log - `PackageState` entity missing';
```

Comment:
I dislike revealing our datastore schema internals in a public facing endpoint.

Reply:
how about - `no tracking information`?

Reply:
Much better!